kitsune/wiki/views.py

```diff
@@ -478,7 +478,7 @@ def edit_init_and_perms(request, document_slug, revision_id=None, doctype="doc")
         if not doc.allows(user, "create_revision"):
             raise PermissionDenied
     if doctype == "meta":
-        if not doc.allows(user, "edit") or not user.is_staff:
+        if (not doc.allows(user, "edit") or not user.is_staff):
```

Comment:
The logic, as it is, is definitely wrong, but I think you meant the following instead:
```suggestion
        if not (doc.allows(user, "edit") or user.is_staff):
```
or you could do it this way too:
```suggestion
        if not doc.allows(user, "edit") and not user.is_staff):
```